### PR TITLE
Update x.ps1

### PR DESCRIPTION
**Pull request by Aksh-Raizada**
-Used a more descriptive variable name for $xpy_args ($xpyArgs or $pythonScriptArgs is more descriptive.) 
-Added a comment to the Invoke-Application() function to explain why the $handle variable is cached. 
-Used a more descriptive error message in the <span class="math-inline">msg\ variable. 
-Added a link to the Python documentation in the help: section of the error message.

### CODE REVIEW

File: x.ps1 [hunk 1]
```diff
@@ -7,57 +7,58 @@ $ErrorActionPreference = "Stop"
 # syntax check
 Get-Command -syntax ${PSCommandPath} >$null
 
-$xpy = Join-Path $PSScriptRoot x.py
-# Start-Process for some reason splits arguments on spaces. (Isn't powershell supposed to be simpler than bash?)
-# Double-quote all the arguments so it doesn't do that.
-$xpy_args = @("""$xpy""")
+$pythonScriptArgs = @("$($PSScriptRoot)\x.py")
```
Comment (kadiwa4) [inline x.ps1:10]: Does that work when there's a space in the value of `$PSScriptRoot`?
And why did you remove the comment?

**Review by kadiwa4 — COMMENTED** [review]
The comment in `Invoke-Application` and the Python link were already there.

**Comment by ChrisDenton** [comment]
Why change the number of spaces to indent by? This seems like unnecessary churn and obscures the actual changes.

File: x.ps1 [hunk 2]
```diff
@@ -7,57 +7,58 @@ $ErrorActionPreference = "Stop"
...
 $found = (Get-Application "python*" | Where-Object {$_.name -match '^python[2-3]\.[0-9]+(\.exe)?$'})
 if (($null -ne $found) -and ($found.Length -ge 1)) {
-    $python = $found[0]
-    Invoke-Application $python $xpy_args
+  $python = $found[0]
+  Invoke-Application $python $pythonScriptArgs
 }
 
-$msg = "${PSCommandPath}: error: did not find python installed`n"
+$msg = "${PSCommandPath}: error: could not find a compatible Python installation`n"
```
Comment (ChrisDenton) [inline x.ps1:61]: This just seems like a longer way to say the exact same thing, no? Personally I'd prefer to keep the succinct version unless there's something more actionable to add.

**Comment by ChrisDenton** [comment]
Closing due to lack of activity. If you'd like to continue working on this please reopen *before* pushing any new commits.
